src/jsutils/__tests__/symbolSupport-test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { hasSymbolSupport, applyToStringTag } from '../symbolSupport';
+
+describe('symbolSupportTests', () => {
+  // NOTE Symbol appeared in nodejs in 0.12.18 but was largely unusable in
+  // that format. Symbol.toStringTag showed up in node 6.4.0, but was available
+  // behind a flag as early as 4.9.1.
+  const [, major, minor, patch] = /^v(\d+)\.?(\d+)?\.?(\d+)?/.exec(
+    process.version,
+  );
+
+  it('should have Symbol in scope if the version >= 4.9.1', () => {
+    expect(major >= 4).to.equal(true);
+
+    if (major === 4) {
+      expect(minor >= 9).to.equal(true);
+    }
+
+    if (minor === 9) {
+      expect(patch >= 1).to.equal(true);
+    }
+
+    expect(hasSymbolSupport()).to.equal(true);
+  });
+
+  it('should have Symbol in scope if the version >= 6.4.0', () => {
```

Comment:
I checked and looks like it supported since `6.0.0`:
https://node.green/#ES2015-built-ins-well-known-symbols-Symbol-toStringTag
![image](https://user-images.githubusercontent.com/8336157/40581391-cff1cc94-615f-11e8-8492-a37ad0eefeb9.png)

------------------------
Given that `package.json` requires `>=6.0.0`:
https://github.com/graphql/graphql-js/blob/0a30b62964efe16e3e87b0e8fea40bc9665f9393/package.json#L21
Please remove two above tests.